_posts/2016-02-17-2016-roadmap.markdown

```diff
@@ -0,0 +1,100 @@
+
+### Building Applications with Polymer  
+
+We see [Progressive Web Apps](https://developers.google.com/web/progressive-web-apps?hl=en) as a big part of the future of the web - web applications that take [“all the right vitamins”](https://infrequently.org/2015/06/progressive-apps-escaping-tabs-without-losing-our-soul/) to be added to a user’s homescreen, work while the user is offline, send a user push notifications, and more. Many of these types of features that [Service Worker](http://www.html5rocks.com/en/tutorials/service-worker/introduction/) provides also marry extremely well with a component-driven, tight-to-the-platform development model supported by Web Components. We want to publish more around how we see these pieces playing together - how to build on the web platform the way it is intended - and also provide a suite of tools to actually manifest these ideas.
+
+*Goals*:
+
+- Prototype all the pieces needed to build a complete [Progressive] Web App with Web Components and Service Worker
+  - Publish philosophy around how to structure a web component-based progressive web app - building on the web the way the platform intended!
+  - Ship a routing solution
+  - Ship a lazy-loading solution
+  - Ship a l10n/i18n solution
+
+### Element Maintenance and Performance
+Much of the core Polymer team’s time is spent fixing issues on existing Polymer elements. We’re incredibly eager to push forward with new elements to solve new problems, but need to get a handle on the existing maintenance burden. We're approaching this from multiple angles - continually improving quality and fixing bugs, improving the core Polymer team's tools and systems for maintenance, and aiming to be better contributors to the open web components ecosystem.
+
+We're extremely bullish about this growing web component ecosystem, and excited to help it grow in any way we can. We’ll lay out in a follow-up blog post many of the ways we’re hoping to make it easier and clearer to contribute to the Polymer project, improve turnaround from within the Polymer core team, and better promote community-built web components.
+
+*Goals*:
+
+- Reduce and achieve sustainable element maintenance costs
+  - Help promote community-driven elements and bring in additional maintainers from the community.
+  - Increase number of merged community Pull Requests to elements by 100%
+  - Keep time-to-triage for incoming element issues < 5 days.
+
+We’re also always looking for opportunities to improve element performance. We’ll particularly be taking a hard look at the most commonly-used elements - things like [paper-input](https://elements.polymer-project.org/elements/paper-input) and [iron-icon](https://elements.polymer-project.org/elements/iron-icon) - to make them as fast and lightweight as possible.
```

Comment:
I think we could mention that elements that are used many times on the page (input, button, icon) are our the first targets. One should not be penalized for choosing paper-input over input for example.